### PR TITLE
New package: AMD.AMDSoftwareCloudEdition version 23.Q3

### LINKED ISSUE #166690: [Package Request]: AMD Software: Cloud Edition
### How can we help?

I would like help so I can submit the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://learn.microsoft.com/en-us/azure/virtual-machines/windows/n-series-amd-driver-setup
Publisher: Advanced Micro Devices, Inc.
Package Name: AMD Software: Cloud Edition
Description: AMD Software: Cloud Edition is a specialized package containing AMD software designed for cloud environments. The AMD Software: Cloud Edition drivers must be installed to take advantage of the GPU capabilities of Azure NGads V620 Series VMs.
Package Version: 23.Q3
Installer URL: https://go.microsoft.com/fwlink/?linkid=2248541

### PR DESCRIPTION
**Pull request by jbne**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/166689)

resolve #166690

### CODE REVIEW

File: manifests/a/AMD/AMDSoftwareCloudEdition/23.Q3/AMD.AMDSoftwareCloudEdition.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+PackageVersion: 23.Q3
+Scope: machine
+InstallModes:
+- silent
+- interactive
+InstallerSwitches:
+  Silent: -install
+  SilentWithProgress: -install
+  Log: -log <LOGPATH>
+Installers:
```
Comment (stephengillie) [inline manifests/a/AMD/AMDSoftwareCloudEdition/23.Q3/AMD.AMDSoftwareCloudEdition.installer.yaml:14]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @jbne,

It seems like `VCRedist` is a dependency:
image
image

**Comment by stephengillie** [comment]
Hi @jbnz,

Verify:
image

Since the package writes `1.00.0000` to the Registry, I believe a `DisplayVersion` is needed here, to help the package manager to match the Registry data to this manifest. What do you think?

**Comment by jbne** [comment]
@stephengillie I think that would be proper, thanks for pointing that out!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**